frontend/src/components/transactions/__tests__/__snapshots__/ActionRow.test.tsx.snap

```diff
@@ -29,7 +29,7 @@ Array [
             <div
               className="action-row-title col"
             >
-              Called method: 'on_staking_pool_withdraw' in contract: 
+              component.transactions.ActionMessage.FunctionCall.called_method
```

Comment:
I believe that using "cimode" of `i18next` (which make all translation requests evaluate to just keys) helps in unit testing, but is bad for e2e testing.
@frol @shelegdmitriy Need your opinion :)

Reply:
> Arguably, I can see benefit of reviewing how all the translations affect the test and whether all of them actually render proper result, so we could even consider having 1 + N snapshots (1 with raw translation keys, and N snapshots for each supported language)

I like that idea. I'll implement N more copies tests later then.

Reply:
Well, for me it's fine that snapshot store the proper translation code. Even from contributor's side it's much easier to follow by than to see the translation in english (default one). LGTM